app/src/main/java/com/github/se/eventradar/ui/hosting/Hosting.kt

```diff
@@ -0,0 +1,281 @@
+import com.github.se.eventradar.model.event.Event
+import com.github.se.eventradar.model.event.EventCategory
+import com.github.se.eventradar.model.event.EventTicket
+import com.github.se.eventradar.ui.BottomNavigationMenu
+import com.github.se.eventradar.ui.home.EventList
+import com.github.se.eventradar.ui.navigation.NavigationActions
+import com.github.se.eventradar.ui.navigation.TOP_LEVEL_DESTINATIONS
+import java.time.LocalDateTime
+
+@Composable
+fun HostingScreen(navigationActions: NavigationActions) {
+  val mockEvents =
+      listOf(
+          Event(
+              "NYE2025",
+              "User1", // todo
+              LocalDateTime.MIN,
+              LocalDateTime.MAX,
+              Location(83.39, 2.992, "EPFL"),
+              "enjoy your time on the dacefloor",
+              EventTicket("Standard", 0.0, 500),
+              "jg@joytigoel.com",
+              mutableSetOf("2989jdgj23", "32923jkbd23"),
+              mutableSetOf("20982jwdwk", "j1ou1e]d8223"),
+              EventCategory.MUSIC,
+              "89379"),
+          Event(
+              "NYE2026",
+              "User2", // todo
+              LocalDateTime.now(),
+              LocalDateTime.MAX,
+              Location(83.49, 56.992, "161 makepeace avenue, n666es"),
+              "Forget and Enjoy",
+              EventTicket("regular", 0.0, 10000),
+              "valerian@joytigoel.com",
+              mutableSetOf("298jhk", "jwj8223"),
+              mutableSetOf("20982jhk", "j1ou1e8223"),
+              EventCategory.SPORTS,
+              "89298"),
+          Event(
+              "NYE2027",
+              "User3", // todo
+              LocalDateTime.now(),
+              LocalDateTime.MIN,
+              Location(83.39, 66.992, "161 makepeace avenue, n666es"),
+              "Join the Community",
+              EventTicket("regular", 0.0, 10000),
+              "valerian@joytigoel.com",
+              mutableSetOf("298jhk", "jwj8223"),
+              mutableSetOf("20982e2hk", "j1ou223e8223"),
+              EventCategory.COMMUNITY,
+              "89298"))
+
+  var selectedTabIndex by remember { mutableIntStateOf(0) }
+  val context = LocalContext.current
+
+  ConstraintLayout(modifier = Modifier.fillMaxSize().testTag("hostingScreen")) {
+    val (logo, tabs, eventList, bottomNav, buttons) = createRefs()
+    Row(
+        modifier =
+            Modifier.fillMaxWidth()
+                .constrainAs(logo) {
+                  top.linkTo(parent.top, margin = 32.dp)
+                  start.linkTo(parent.start, margin = 16.dp)
+                }
+                .testTag("logo"),
+        verticalAlignment = Alignment.CenterVertically) {
+          Image(
+              painter = painterResource(id = R.drawable.event_logo),
+              contentDescription = "Event Radar Logo",
+              modifier = Modifier.size(width = 186.dp, height = 50.dp))
+        }
+
+    TabRow(
+        selectedTabIndex = selectedTabIndex,
```

Comment:
We are using TabRows already in the Home screen. Maybe you could create a component out of these tabs in order not to repeat a large amount of code across the app?

Reply:
Additionally, if you don't plan on using a second tab (there's currently one), the `TabRow` + `Tab` should be deleted entirely